### DISCUSSION TITLE
Missing context menu

### DISCUSSION THREAD

**Discussion by uk1988** [coderaiser/cloudcmd#392] [Q&A]
Hej

I am new to the cloudcmd :)
We have written helm chart to use cloudcmd in our little project. 

Helm chart:
[file-browser.zip](https://github.com/coderaiser/cloudcmd/files/11322431/file-browser.zip)

template file:
[values.file-browser.yaml.zip](https://github.com/coderaiser/cloudcmd/files/11322438/values.file-browser.yaml.zip)

All works perfectly with the exception of the context menu. 
At some point, it stopped having the context menu available on files. I can not create new files or edit them in any way I can only download them.
What am I doing wrong? 

the /opt/cloudcmd/config.json:
{
   "configDialog":true,
   "console":false,
   "contact":true,
   "log":false,
   "name":"File browser",
   "oneFilePanel":true,
   "prefix":"/files",
   "root":"/opt/adbsg",
   "terminal":false
}
example of deployment: 
[cloudcmd.txt](https://github.com/coderaiser/cloudcmd/files/11322474/cloudcmd.txt)

**Comment by coderaiser** [comment]
Do you have any errors in console of your browser?

**Reply by uk1988** [reply]
Hej yes I do

![image](https://user-images.githubusercontent.com/66792788/234814837-e4a1b15c-dde9-40a2-bde6-50107a9aa9c6.png)

For ingress, we have a self-signed cert that is not trusted.

**Reply by coderaiser** [reply]
Try to generate certificate with [letsencrypt](https://letsencrypt.org/)

**Reply by uk1988** [reply]
will do, you think the issue will go away if I put a trusted cert in place?

**Reply by coderaiser** [reply]
I see no other errors, so let's start from this
